api/src/main/java/org/apache/cloudstack/api/response/VolumeResponse.java

```diff
@@ -288,6 +289,14 @@ public class VolumeResponse extends BaseResponseWithTagInformation implements Co
     @Param(description = "volume uuid that is given by virtualisation provider (only for VMware)")
     private String externalUuid;
 
+    @SerializedName(ApiConstants.VOLUME_CHECK_RESULT)
+    @Param(description = "details for the volume check result, they may vary for different hypervisors, since = 4.19.1")
```

Comment:
@harikrishna-patnala can you confirm the release version, if it's targeted for 4.19.1, rebase with 4.19 branch

Reply:
Thanks @sureshanaparti I've rebased the branch to 4.19.

Reply:
ok, thanks @harikrishna-patnala